### ISSUE TITLE
Nightly CI is logging huge amounts of unit testing logs

### ISSUE THREAD

**Issue by mikalstill** [shakenfist/shakenfist#529]
Like 15mb of logs specifically.

**Comment by mikalstill** [comment]
Hmmm, its pretty verbose on my laptop too... 5,000 lines of output.

**Comment by mandoonandy** [comment]
It is big, but I've found it very useful. Admittedly it does a bit of effort to work out how to navigate successfully.

**Comment by mikalstill** [comment]
Yeah, I think this is because of 6c8c1b8e8822bd782107aff7f57080f953e28f42. I think the problem is that its hard to see which unit test a given piece of debugging output relates to.

**Comment by mikalstill** [comment]
Closing during a triage sweep of old issues: the nightly CI pipeline this described (and its 15MB unit-test logs) no longer exists — CI has been rebuilt around GitHub Actions with self-hosted runners and a merge queue. Log-volume concerns with the current CI would be a new issue.

*(Triage assisted by Claude Code; closure approved by Michael Still.)*
